toolchain/sem_ir/typed_insts.h

```diff
@@ -189,6 +189,15 @@ struct AssociatedEntityType {
   TypeId entity_type_id;
 };
 
+// An adapted type declaration in a class, of the form `adapt T;`.
+struct AdaptDecl {
```

Comment:
Order relative to other declarations? (before Assoc*?)

Reply:
Done.